File: ClientApp/app/models/persistency/cardBad.ts
import {User} from './user';
import {CardBase} from './cardBase';

export enum BadVoteType {
    Facile = 0,
    Sentito = 1,
    Inaspettato = 2
}

export class BadVote {
    Type: BadVoteType;
    User: User;
}

export class CardBad extends CardBase {
    Votes: BadVote[];
}

File: BusinessLogic/Models/Persistency/CardBad.cs
using System.Collections.Generic;

namespace kRetro.BusinessLogic.Models.Persistency
{
    public enum BadVoteType
    {
        Easy = 0,
        Significant = 1,
        Unexpected = 2
    }

    public class BadVote
    {
        public BadVoteType Type {get;set;}
        public User User {get;set;}
    }

    public class CardBad : CardBase
    {
        public CardBad() : base(CardType.Bad)
        {
        }

        public List<BadVote> Votes {get;set;} = new List<BadVote>();
    }
}